.drone.yml

```diff
@@ -48,8 +49,24 @@ pipeline:
     commands:
       - docker login -u="ukhomeofficedigital+hocs" -p=$${DOCKER_PASSWORD} quay.io
       - docker tag hocs-info-service quay.io/ukhomeofficedigital/hocs-info-service:build-$${DRONE_BUILD_NUMBER}
-      - docker tag hocs-info-service quay.io/ukhomeofficedigital/hocs-info-service:latest
+      - export BRANCH_NAME_TAGFRIENDLY=`echo "$${DRONE_COMMIT_BRANCH}" | sed 's$/$_$g'`
+      - docker tag hocs-info-service quay.io/ukhomeofficedigital/hocs-info-service:branch-$${BRANCH_NAME_TAGFRIENDLY}
       - docker push quay.io/ukhomeofficedigital/hocs-info-service:build-$${DRONE_BUILD_NUMBER}
+      - docker push quay.io/ukhomeofficedigital/hocs-info-service:branch-$${BRANCH_NAME_TAGFRIENDLY}
+    when:
+      branch: [master, epic/*]
```

Comment:
Should we remove `master` here? As if it's master we want `latest` which is handled by the `install-docker-image-latest:` step?

Reply:
Well, we also want it to tag `build-XXX`, so we may as well keep it in this step.
Also makes sense to push `branch-master`, just for consistency.